src/test/java/hw8/pages/forms/MetalsColorsForm.java

```diff
@@ -0,0 +1,67 @@
+package hw8.pages.forms;
+
+import com.epam.jdi.light.elements.common.Label;
+import com.epam.jdi.light.elements.complex.JList;
+import com.epam.jdi.light.elements.complex.dropdown.Dropdown;
+import com.epam.jdi.light.elements.composite.Form;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.Css;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.JDropdown;
+import com.epam.jdi.light.ui.html.elements.common.Button;
+import hw8.data.beans.DataBean;
+import hw8.pages.custom.MultiDropdown;
+
+import java.util.Arrays;
+
+public class MetalsColorsForm extends Form<DataBean> {
+
+    @Css("#summary-block .info-panel-section label")
+    public JList<Label> summary;
+
+    @Css("#elements-checklist label")
+    public JList<Label> elements;
+
+    @JDropdown(
+            root = "div[ui='dropdown']",
+            value = ".filter-option",
+            list = "li",
+            expand = ".caret")
+    public Dropdown colors;
+
+    @JDropdown(
+            root = "div[ui='combobox']",
+            value = "input",
+            list = "li",
+            expand = ".caret")
+    public Dropdown metals;
+
+    @JDropdown(
+            root = "div[ui='droplist']",
+            value = "label",
+            list = "li",
+            expand = ".caret")
+    public MultiDropdown vegetables;
+
+    @Css("button#submit-button")
+    public Button submit;
+
+    @Override
+    public void fill(DataBean entity) {
```

Comment:
What kind of Data Bean is this ? The name of the class should be selected with reference to it business purpose.

Reply:
Refactored.